### DISCUSSION TITLE
Only a single instance is allowed.

### DISCUSSION THREAD

**Discussion by Coedy** [fuatakgun/eufy_security#622] [General]
Trying to get this set up today. Managed to get further than previously. Get to the Captcha section and it takes a very long time to load before popping up:
Eufy Security
Only a single instance is allowed.
I've tried removing the integration (delete) and trying again as well as a reboot or two between each action with no luck.

**Comment by fuatakgun** [comment]
But what happens after only one instance is allowed window?

In any case, enable debug logs for the integration and share them over here

**Reply by Coedy** [reply]
It displays an 'OK' button, when you click that the integration screen shows the previous message of:
"Failed to set up: Warning: Captcha required - Go to Configurations page for Eufy Security Integration to enter the code"
However you are unable to make it display the 'reconfigure' button to display the captcha until you remove the integration and try setting it up again.

How do you enable Debug on a specific integration?

**Reply by Coedy** [reply]
@fuatakgun 
I enabled Debugging for the integration (much easier than logger!) but only seeing the following in the logs:

2023-01-01 18:22:19.470 WARNING (MainThread) [homeassistant.config_entries] Config entry 'localhost' for eufy_security integration could not authenticate: Warning: Captcha required - Go to Configurations page for Eufy Security Integration to enter the code
2023-01-01 18:23:44.124 WARNING (MainThread) [homeassistant.config_entries] Config entry 'localhost' for eufy_security integration could not authenticate: Warning: Captcha required - Go to Configurations page for Eufy Security Integration to enter the code

It won't allow me to turn on debug for the 'attention required' integration.

**Reply by fuatakgun** [reply]
Because debugger is not supported yet, you need to enable logger

**Reply by Coedy** [reply]
Feel free to close this. While waiting for my home assistant to restart after enabling logger I was playing with my phone and say 30emails. I had 2FA enabled. I disabled it, left it for 5mins or so. rebooted again, added everything from scratch and it worked.

**Reply by fuatakgun** [reply]
But did not you receive mfa request in home assistant rather than captcha?

**Reply by Coedy** [reply]
No, I had the captcha request. then it did as I described earlier.
I disabled 2FA and then it worked.
